### ISSUE TITLE
Better translation platform

### ISSUE THREAD

**Issue by comradekingu** [ProtonMail/WebClients#180]
Hello, I'd like to help the translation, but Crowdin is a non-starter with all the user tracking and profiling it does. It is also a lackluster tool in terms of actually facilitating translation.

It would be nice to see ProtonMail over at Hosted Weblate, which actually is (optionally) gratis for libre software, so in every respect quite the opposite of what Crowdin is.

Self-hosting Weblate is an option, but [Hosted Weblate](https://weblate.org) would be preferable, seeing as there are already a lof of e-mail related software on there. Send a request and add the project and I can help administrate, if you add "kingu" as an admin, but in any event I can fix failing checks and contribute in the languages I know already.

**Comment by SchnWalter** [comment]
What's your game? I've been seeing you in quite a few projects that I follow. You clearly don't "_like to help the translation_".

https://github.com/LibrePCB/LibrePCB/issues/615

P.S. Using obscure tracking methods doesn't mean you are not tracking. It's actually worst.

**Comment by comradekingu** [comment]
You have seen me translate them, or you have seen me ask or help move them off either Crowdin or Transifex? I translate all the projects that move over, no small feat.
For e-mail related software, I have translated Mailpile, Mailvelope, Thunderbird, GNU Mailman (not done), GPG, elemantaryOS Mail, Enigmail, keys.openpgp.org, K-9 Mail etc.

There is nothing obscure about the tracking in Crowdin, other than how it is described in the terms and conditions. It uses the biggest names in that business, which I argue is far worse.
Hosted Weblate uses Matomo, inhouse, which is a libre software web analytics platform. The scrapers I checked were excluded from the "hosted." subdomain.
What is the tracking involved in self-hosting Weblate? The only thing I can think of is Gravatars, which are cached to mitigate said problem, knowing the service is used by a user is some level of detail, for those that use it. What is preferable to Weblate?
